pkg/kubectl/describe.go

```diff
@@ -1801,7 +1833,7 @@ func describeServiceAccount(serviceAccount *api.ServiceAccount, tokens []api.Sec
 			tokenSecretNames = []string{}
 		)
 
-		for _, s := range serviceAccount.ImagePullSecrets {
+		for _, s := range saPullSecrets {
```

Comment:
@liggitt thanks, this makes sense. Maybe describe could output something like:
```
$ oc describe sa/test
Name:           test
Namespace:      default
Labels:         <none>

Image pull secrets:     test-dockercfg-1fgip
                        my-pull-secret (inaccessible)

Mountable secrets:      test-token-t0tym
                        test-dockercfg-1fgip

Tokens:                 test-token-ai8bu
                        test-token-t0tym
```

This way, we don't mark a pull secret as `(deleted)` if it still exists but the user cannot list it. WDYT?